README.md

```diff
@@ -52,6 +52,7 @@ interface Options {
   baseUrl?: string
   eventUrl?: string
   eventsSyncInterval?: number
+  pollingInterval?: number
```

Comment:
I'd add a `pollingEnabled` as well. It would also be helpful to add a section explaining Polling and Streaming, detailing how they're used and configured.

Reply:
Done - now polling and streaming is opt in. As we discussed: "Streaming takes precedence, but falls back to polling unless disabled"